src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -72,7 +77,14 @@ public VectorLoglessPairHMM(Implementation implementation, PairHMMNativeArgument
                 if (!isSupported) {
                     throw new UserException.HardwareFeatureException("Machine does not support OpenMP AVX PairHMM.");
                 }
-                logger.info("Maximum number of OpenMP threads for AVX : " + args.maxNumberOfThreads);
```

Comment:
We removed this because the latest version of GKL (0.5.2) now outputs this information, along with the total number of available threads/cores. It also outputs a warning when more threads are requested than are available.